ddcornwall/nara-alaskana
# Refactor based on pjs

Closes #23 , closes #10, closes #13 



- Changed function calls on buttons to handlers
- Moved script to bottom of html pages
- Put handlers code towards top of search.js
- Eliminated browse js


QA:
- [Check all browse buttons ] 
- [Do a search and page through results till end ]
